modules/ROOT/pages/batch-processing-concept.adoc

```diff
@@ -186,6 +191,35 @@ Every processed record of the Batch Job Instance starts with the same variables
 In the On Complete phase, none of these variables (not event the original ones) are visible. Only the final result is available in this phase. Moreover, since the Batch Job Instance executes asynchronously from the rest of the flow, no variable set in either a Batch Step or the On Complete phase will be visible outside the Batch Scope.
 
 
+=== Scheduling Strategy
+
+A Mule application can contain several batch job definitions. Each definition 
+has its own scheduling strategy. The Batch Job scope provides 
```

Comment:
`The Batch Job scope provides  *Scheduling Strategy* options for controlling the execution of job instances: ` -> `Scheduling Strategy allows controlling how instances of a given Batch Job are executed`

Reply:
done